Index: pyabiword-0.8.0/configure.in
===================================================================
--- pyabiword-0.8.0.orig/configure.in	2011-11-08 15:33:28.279267476 +0100
+++ pyabiword-0.8.0/configure.in	2011-11-08 15:34:00.750859252 +0100
@@ -17,7 +17,7 @@
 AM_PROG_LIBTOOL
 AC_C_CONST
 
-abi_pkg='abiword-2.8 >= 2.8.0'
+abi_pkg='abiword-2.9 >= 2.9.0'
 PKG_CHECK_MODULES(ABIWORD, "$abi_pkg")
 AC_SUBST(ABIWORD_CFLAGS)
 AC_SUBST(ABIWORD_LIBS)
